### PR TITLE
refactor(audioclipimage): upgrade jina and code style

### PR DESCRIPTION
**Pull request by bwanglzu**

### CODE REVIEW

File: jinahub/encoders/image/AudioCLIPImageEncoder/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM jinaai/jina:2.0.10-perf
+FROM jinaai/jina:2.0.23-perf
```
Comment (tadejsv) [inline jinahub/encoders/image/AudioCLIPImageEncoder/Dockerfile:1]: The whole file should be removed

File: jinahub/encoders/image/AudioCLIPImageEncoder/tests/requirements.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
-jina[http]==2.0.0rc9
+jina[http]==2.0.23
```
Comment (tadejsv) [inline jinahub/encoders/image/AudioCLIPImageEncoder/tests/requirements.txt:1]: Should be removed

**Comment by jakob1111996** [comment]
This refactoring is done already in another branch, see #161 
We can therefore close this PR.

**Comment by bwanglzu** [comment]
ok I'll close this one, thanks for reminding @jakob1111996

**Pull request closed without merging**